### DISCUSSION TITLE
ABP Community Talks 2023.8: What’s coming with .NET 8.0 & ABP v8.0

### DISCUSSION THREAD

**Discussion by bigebesikci** [abpframework/abp#18076] [Show and tell]
Hi ABP Community,

I came with a new episode of ABP Community Talks! 🙌

In the ABP Community Talks 2023.8, we will talk about the .NET and ABP's version 8.0 and what is coming with these new versions.

Make sure to catch the live event **on November 23, 2023 at 17:00 UTC**!
Raffles for giveaways will take place in the live event. 🎁

Register from 👉 https://kommunity.com/volosoft/events/abp-community-talks-20238-whats-coming-with-net-80-abp-v80-a196d644

See you then!

**Comment by hikalkan** [comment]
Steve Sanderson ( @SteveSandersonMs ) will be our guest in this community talk. You will be able to ask your questions to the creator of Blazor in live QA part. 

![image](https://github.com/abpframework/abp/assets/1210527/af3cd1da-9c26-45a9-b7a6-eab0842c6070)

Register here: https://kommunity.com/volosoft/events/abp-community-talks-20238-whats-coming-with-net-80-abp-v80-a196d644
